tests/Schema/DataObject/Plugin/FiltersTest.php

```diff
@@ -25,80 +25,14 @@ class FiltersTest extends SapphireTest
         'null' => null
     ];
 
-    public function testFilterArguments()
-    {
-        // Contains
-        $this->testFilter(
-            new ContainsFilter(),
-            'contains',
-            array_slice($this->values, 0, 2, true)
-        );
-
-        // Endswith
-        $this->testFilter(
-            new EndsWithFilter(),
-            'endswith',
-            array_slice($this->values, 0, 2, true)
-        );
-
-        // EqualToFilter
-        $this->testFilter(
-            new EqualToFilter(),
-            'eq',
-            $this->values
-        );
-
-        // GreaterThanFilter
-        $this->testFilter(
-            new GreaterThanFilter(),
-            'gt',
-            $this->values
-        );
-
-        // GreaterThanFilter
-        $this->testFilter(
-            new GreaterThanOrEqualFilter(),
-            'gte',
-            $this->values
-        );
-
-        // InFilter
-        $this->testFilter(
-            new InFilter(),
-            'in',
-            $this->values['array']
-        );
-
-        // LessThanFilter
-        $this->testFilter(
-            new LessThanFilter(),
-            'lt',
-            $this->values
-        );
-
-        // LessThanFilter
-        $this->testFilter(
-            new LessThanOrEqualFilter(),
-            'lte',
-            $this->values
-        );
-
-        // NotEqualFilter
-        $this->testFilter(
-            new NotEqualFilter(),
-            'ne',
-            $this->values
-        );
-
-        // StartsWithFilter
-        $this->testFilter(
-            new StartsWithFilter(),
-            'startswith',
-            array_slice($this->values, 0, 2, true)
-        );
-    }
-
-    private function testFilter(FieldFilterInterface $filter, string $identifier, array $params)
+    /**
+     * @param FieldFilterInterface $filter
+     * @param string $identifier
+     * @param array $params
+     * @dataProvider filterProvider
+     * @return void
```

Comment:
```suggestion
     * @dataProvider filterProvider
```
No need to have PHPDocs that duplicate the typings - these have a tendency to get out of date and since we have typehints they don't provide any extra information.